### PR TITLE
NEST-339: Add feature to make build number visible after deployment

### PR DESCRIPTION
**Pull request by Piedone**
[NEST-339](https://lombiq.atlassian.net/browse/NEST-339)

[NEST-339]: https://lombiq.atlassian.net/browse/NEST-339?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by wAsnk — APPROVED** [review]

**Comment by jsoref** [comment]
I _think_ your workflow is using:
https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/3847653530/workflow#L41-L47

I think you'd be able to "fix" this with a second job, admittedly, you'd want to define a reusable workflow for it and have it fill in the other bits based on the first one.

I looked into trying to write a reusable workflow to make this stuff clean and the effort made me shrug them off because of the amount of duplication it required...

**Review by wAsnk — APPROVED** [review]

File: Readme.md [hunk 1]
```diff
@@ -25,73 +25,74 @@ This also serves as an example of an ASP.NET Core web app using Orchard from NuG
...
+  - [`Lombiq.HelpfulLibraries.Samples`](https://github.com/Lombiq/Helpful-Libraries/tree/dev/Lombiq.HelpfulLibraries.Samples): Example Orchard Core module that makes use of Lombiq Helpful Libraries.
+  - [`Lombiq.Hosting.Azure.ApplicationInsights`](https://github.com/Lombiq/Orchard-Azure-Application-Insights): This [Orchard Core](https://www.orchardcore.net/) module enables easy integration of [Azure Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) telemetry into Orchard. Just install the module, configure the instrumentation key from a configuration source (like the _appsettings.json_ file) as normally for AI, and collected data will start appearing in the Azure Portal. As seen on [the Orchard community meeting](https://www.youtube.com/watch?v=NKKR4R3UPog). Note that this module has an Orchard 1 version in the [dev-orchard-1 branch`](https://github.com/Lombiq/Orchard-Azure-Application-Insights/tree/dev-orchard-1). Note that the module depends on [Helpful Libraries](https://github.com/Lombiq/Helpful-Libraries/).
```
Comment (0liver) [inline Readme.md:42]: @Piedone Please revisit the formatting of links in this file - there are quite a few trailing `` ` `` characters in link texts, and at least one link that's not a technical identifier but formatted as one:

image

Comment (Piedone) [reply Readme.md:42]: Thanks for letting me know, fixed them.

Comment (0liver) [reply Readme.md:42]: You're welcome.

**Pull request merged**